src/blocks/google-map/settings.js

```diff
@@ -0,0 +1,577 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+
+const { InspectorControls } = wp.blockEditor;
```

Comment:
```suggestion
import { InspectorControls } from '@wordpress/block-editor';
```

This will also probably needed to be added as a dependency in package.json

Reply:
IMO, Gutenberg itself adds this in `wp. blockEditor` object hence we do not need to add the dependency for it ideally.